build/secondary/third_party/android_tools/BUILD.gn

```diff
@@ -28,21 +28,13 @@ android_java_prebuilt("uiautomator_java") {
   jar_path = "$android_sdk/uiautomator.jar"
 }
 
-android_resources("android_support_design_resources") {
-  v14_skip = true
-  resource_dirs = [ "$android_sdk_root/extras/android/support/design/res" ]
-  deps = [
-    ":android_support_v7_appcompat_resources",
-  ]
-  custom_package = "android.support.design"
+android_java_prebuilt("android_support_annotations_javalib") {
+  jar_path = "$android_sdk_root/extras/android/support/annotations/android-support-annotations.jar"
 }
 
-android_java_prebuilt("android_support_design_java") {
-  deps = [
-    ":android_support_v7_appcompat_java",
-    ":android_support_design_resources",
-  ]
-  jar_path = "$android_sdk_root/extras/android/support/design/libs/android-support-design.jar"
+java_prebuilt("android_support_multidex_java") {
+  #supports_android = true
```

Comment:
Nope, I'll remove the line.

Our build fails with it, because it's not used by our build system.